helm/templates/promtail/deployment.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: extensions/v1beta1
```

Comment:
Promtail is deployed as a DaemonSet. I think you should update the filename so that it matches the Kubernetes resource.

Reply:
good catch